website/src/components/Screengrabs/atmos-validate-editorconfig--help.html

```diff
@@ -0,0 +1,389 @@
+
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="UTF-8">
+		<title>terminal-to-html Preview</title>
+		<style>.term-container {
+  background: #171717;
+  border-radius: 5px;
+  color: white;
+  word-break: break-word;
+  overflow-wrap: break-word;
+  font-family: "SFMono-Regular", Monaco, Menlo, Consolas, "Liberation Mono", Courier, monospace;
+  font-size: 12px;
+  line-height: 20px;
+  padding: 14px 18px;
+  white-space: pre-wrap;
+}
+
+.term-container img { max-width: 100%; }
+
+.term-container time { padding-right: 1ex; }
+
+.term a { color: inherit; text-decoration: underline; text-decoration-style: dashed; }
+.term a:hover { color: #2882F9 }
+
+@keyframes blink-animation {
+  to {
+    visibility: hidden;
+  }
+}
+
```

Comment:
This is not the perfect version. I am trying to get the perfect versions like you got.